library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayRecorder.kt

```diff
@@ -48,7 +49,7 @@ internal class SessionReplayRecorder : OnWindowRefreshedCallback, Recorder {
     private val windowInspector: WindowInspector
     private val windowCallbackInterceptor: WindowCallbackInterceptor
     private val sessionReplayLifecycleCallback: LifecycleCallback
-    private val processor: RecordedDataProcessor
+    private val blockingQueueHandler: BlockingQueueHandler
```

Comment:
minor nit: probably we don't need word `blocking` in the `blockingQueueExecutorService` and `blockingQueueHandler` names, because what matters is that it is queue, and the actual implementation doesn't really matter.

Reply:
I'm also renaming all blockingQueue classes/items to make the names less connected to the implementation